Gemfile

```diff
@@ -30,6 +29,13 @@ group :test do
   gem 'simplecov-console',                                          :require => false
   gem 'rack', '~> 1.0',                                             :require => false if RUBY_VERSION < '2.2.2'
   gem 'parallel_tests',                                             :require => false
+
+  # Currently, rspec-puppet versions > 2.6.11 break spec tests for Puppet 4
+  if ENV['PUPPET_VERSION'] == '~> 4.0'
+    gem 'rspec-puppet', '<= 2.6.11', :require => false
+  else
+    gem 'rspec-puppet', '~> 2.5', :require => false
+  end
```

Comment:
This is something that should be addressed at the [modulesync](https://github.com/voxpupuli/modulesync_config) level rather than here as our modulesync configs manages our module gemfile

Additionally, this does not take into account Puppet 5.x testing. Puppet 4 will be End of Life this year, so ensuring we have tests for both Puppet 4 and 5 is critical.